app/screens/custom_status/custom_status_modal.tsx

```diff
@@ -130,9 +136,8 @@ class CustomStatusModal extends NavigationComponent<Props, State> {
         this.setState({emoji, text});
     };
 
-    renderRecentCustomStatuses = () => {
+    renderRecentCustomStatuses = (style: any) => {
```

Comment:
Check if the type is `React.CSSProperties`?

Reply:
No actually, the type we are getting is any and taking React.CSSProperties gives error